Explorer/Assets/DCL/Friends/RPCFriendsService.cs

```diff
@@ -0,0 +1,94 @@
+using CommunicationData.URLHelpers;
+using Cysharp.Threading.Tasks;
+using DCL.Profiles;
+using Decentraland.Social.Friendships;
+using System;
+using System.Collections.Generic;
+using System.Net.WebSockets;
+using System.Threading;
+
+namespace DCL.Friends
+{
+    public class RPCFriendsService : IFriendsService
+    {
+        private readonly URLAddress apiUrl;
+        private readonly IFriendsEventBus eventBus;
+        private readonly ClientWebSocket webSocket = new ();
+
+        public RPCFriendsService(URLAddress apiUrl,
+            IFriendsEventBus eventBus)
+        {
+            this.apiUrl = apiUrl;
+            this.eventBus = eventBus;
+        }
+
+        public async UniTask<PaginatedFriendsResult> GetFriendsAsync(int pageNum, int pageSize, CancellationToken ct) =>
+            // TODO
+            new()
+            {
+                Friends = Array.Empty<Profile>(),
+                TotalAmount = 0,
+            };
+
+        public UniTask<bool> IsFriendAsync(string friendId, CancellationToken ct) =>
+            throw new NotImplementedException();
+
+        public async UniTask<PaginatedFriendRequestsResult> GetReceivedFriendRequestsAsync(int pageNum, int pageSize, CancellationToken ct) =>
+            // TODO
+            new()
+            {
+                Requests = Array.Empty<FriendRequest>(),
```

Comment:
When you start the implementation itself consider that `Lists` should be poolable